src/dags/cmsa.py

```diff
@@ -96,7 +96,7 @@
             f"{tmp_dir}/cameras.xlsx",
             f"{tmp_dir}/beacons.csv",
             f"{tmp_dir}/sensors.geojson",
```

Comment:
os.path.join?

Reply:
Oh, that should be addressed as well. Personally I tend to always use the `pathlib` stuff. 

Anyway I'd like to do that in a different PR. The things I addressed in the current PR are things that I came across while working on the "notify source owners on failed imports" feature.

Reply:
Damn you, you triggered my code quality/structure OCD. :stuck_out_tongue:  Anyway fixed.